<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Imperetors" failures="1" tests="1" name="Test" time="3.864" errors="0" timestamp="2025-03-03T09:30:33 EST">
  <testcase classname="testclasses.TC001_Login_page" name="validateLogin" time="0.076">
    <failure type="java.lang.ArrayIndexOutOfBoundsException" message="Index 5 out of bounds for length 5">
      <![CDATA[java.lang.ArrayIndexOutOfBoundsException: Index 5 out of bounds for length 5
at testclasses.TC001_Login_page.validateLogin(TC001_Login_page.java:65)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- validateLogin -->
</testsuite> <!-- Test -->
